src/Card.js

```diff
@@ -0,0 +1,13 @@
+// Each card needs an id, question, and possible answers
+// Data file gives properties of id, question, answers, correct answer
+
+class Card {
+    constructor(id, question, answers, correctAnswer) {
```

Comment:
the parameters match the naming of the properties used for each card object in the prototypeData file.